src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -93,7 +93,10 @@ class="tribe-events-calendar-month__multiday-event-hidden-featured-icon tribe-co
 					></em>
 				<?php endif; ?>
 				<h3 class="tribe-events-calendar-month__multiday-event-hidden-title tribe-common-h8">
-					<?php echo wp_kses_post( get_the_title( $event->ID ) ); ?>
+					<?php
+					// phpcs:ignore
+					echo $event->title;
```

Comment:
https://github.com/moderntribe/the-events-calendar/pull/2987/files?utf8=%E2%9C%93&diff=split&w=1#r366022101